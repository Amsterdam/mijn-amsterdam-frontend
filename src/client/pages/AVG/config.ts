import { AVGRequestFrontend } from '../../../server/services/avg/types';
import { dateSort } from '../../../universal/helpers/date';
import { LinkProps } from '../../../universal/types';
import {
  DisplayProps,
  WithDetailLinkComponent,
} from '../../components/Table/TableV2';

export const displayPropsAanvragen: DisplayProps<
  WithDetailLinkComponent<AVGRequestFrontend>
> = {
  detailLinkComponent: 'Nummer',
  ontvangstDatumFormatted: 'Ontvangen op',
  themas: 'Onderwerp(en)',
};

export const listPageParamKind = {
  inProgress: 'lopende-aanvragen',
  completed: 'afgehandelde-aanvragen',
};

const tableConfigBase = {
  sort: dateSort('dateRequest', 'desc'),
  displayProps: displayPropsAanvragen,
};

export const tableConfig = {
  [listPageParamKind.inProgress]: {
    title: 'Lopende aanvragen',
    filter: (avgVerzoek: AVGRequestFrontend) => !avgVerzoek.datumAfhandeling,
    ...tableConfigBase,
  },
  [listPageParamKind.completed]: {
    title: 'Afgehandelde aanvragen',
    filter: (avgVerzoek: AVGRequestFrontend) => avgVerzoek.datumAfhandeling,
    ...tableConfigBase,
  },
};

export const linkListItems: LinkProps[] = [
  {
    to: 'https://www.amsterdam.nl/privacy/loket/',
    title: 'Loket persoonsgegevens gemeente Amsterdam',
  },
];
