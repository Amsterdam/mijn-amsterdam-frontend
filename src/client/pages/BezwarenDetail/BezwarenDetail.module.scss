@import '../../styles/_global.scss';

.DocumentListHeader {
  display: flex;
  padding-top: 1.6rem;

  > * {
    flex: 1;
    flex-basis: 50%;
  }

  &_Date {
    font-size: var(--ams-heading-level-4-font-size);
    line-height: var(--ams-heading-level-4-line-height);
    font-weight: var(--ams-text-font-weight-bold);
  }
}
