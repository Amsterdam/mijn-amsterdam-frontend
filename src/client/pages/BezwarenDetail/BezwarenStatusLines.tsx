import type { BezwaarStatus } from '../../../server/services/bezwaren/types';
import { EMPTY_UUID } from '../../../universal/helpers/bezwaren';
import { StatusLine } from '../../components';

function transformBezwaarStatusToStatusLines(statussen: BezwaarStatus[]) {
  const index = statussen.findIndex((s) => s.uuid === EMPTY_UUID);
  const activeIndex =
    index > 0 ? index - 1 : index === 0 ? index : statussen.length - 1;

  return statussen.map((status, index) => ({
    id: status.statustoelichting,
    status: status.statustoelichting,
    datePublished: status.datum,
    description: '',
    documents: [],
    isActive: activeIndex === index,
    isChecked: status.uuid !== EMPTY_UUID,
  }));
}

const BezwarenStatusLines = ({
  id,
  statussen,
}: {
  id: string;
  statussen: BezwaarStatus[];
}) => {
  const statusLineItems = transformBezwaarStatusToStatusLines(statussen);

  return (
    <StatusLine
      className=""
      trackCategory="AVG verzoek detail / status"
      items={statusLineItems}
      id={`bezwaar-detail-${id}`}
      documentPathForTracking={(document) =>
        `/downloads/avg-verzoek/${document.title}`
      }
    />
  );
};

export default BezwarenStatusLines;
