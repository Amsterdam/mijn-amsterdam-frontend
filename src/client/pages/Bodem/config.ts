import { generatePath } from 'react-router-dom';

import { LoodMetingFrontend } from '../../../server/services/bodem/types';
import { AppRoutes } from '../../../universal/config/routes';
import { dateSort } from '../../../universal/helpers/date';
import { LinkProps } from '../../../universal/types';

const displayPropsAanvragen = {
  detailLinkComponent: 'Adres',
  datumAanvraagFormatted: 'Aangevraagd',
  status: 'Status',
};

const listPageParamKind = {
  inProgress: 'lopende-aanvragen',
  completed: 'afgehandelde-aanvragen',
};

const tableConfigBase = {
  sort: dateSort('dateRequest', 'desc'),
  displayProps: displayPropsAanvragen,
};

export const tableConfig = {
  [listPageParamKind.inProgress]: {
    ...tableConfigBase,
    title: 'Lopende aanvragen',
    filter: (bodemAanvraag: LoodMetingFrontend) => !bodemAanvraag.processed,
    listPageRoute: generatePath(AppRoutes['BODEM/LIST'], {
      kind: listPageParamKind.inProgress,
    }),
  },
  [listPageParamKind.completed]: {
    ...tableConfigBase,
    title: 'Afgehandelde aanvragen',
    filter: (bodemAanvraag: LoodMetingFrontend) => bodemAanvraag.processed,
    listPageRoute: generatePath(AppRoutes['BODEM/LIST'], {
      kind: listPageParamKind.completed,
    }),
  },
} as const;

export const linkListItems: LinkProps[] = [
  {
    title: 'Meer informatie over lood in de bodem.',
    to: 'https://www.amsterdam.nl/wonen-leefomgeving/bodem/lood-grond/',
  },
] as const;
