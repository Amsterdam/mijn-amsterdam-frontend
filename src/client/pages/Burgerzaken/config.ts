import { generatePath } from 'react-router-dom';

import { AppRoutes } from '../../../universal/config/routes';
import { dateSort } from '../../../universal/helpers/date';
import { IdentiteitsbewijsFrontend, LinkProps } from '../../../universal/types';
import { withOmitDisplayPropsForSmallScreens } from '../../components/Table/helpers';
import {
  DisplayProps,
  WithDetailLinkComponent,
} from '../../components/Table/TableV2';

const displayPropsBase: DisplayProps<
  WithDetailLinkComponent<IdentiteitsbewijsFrontend>
> = {
  detailLinkComponent: 'Type',
  datumAfloopFormatted: 'Geldig tot',
};

const listPageParamKind = {
  identiteitsbewijzen: 'identiteitsbewijzen',
};

const displayProps = withOmitDisplayPropsForSmallScreens(displayPropsBase, [
  'datumAfloopFormatted',
]);

export const tableConfig = {
  [listPageParamKind.identiteitsbewijzen]: {
    title: 'Mijn reisdocumenten',
    displayProps,
    sort: dateSort('datumAfloop', 'desc'),
    listPageRoute: generatePath(AppRoutes['BURGERZAKEN/LIST'], {
      kind: listPageParamKind.identiteitsbewijzen,
    }),
  },
} as const;

export const linkListItems: LinkProps[] = [
  {
    to: 'https://www.amsterdam.nl/burgerzaken',
    title: 'Overzicht en aanvragen bij burgerzaken',
  },
] as const;

export const backLinkDetailPage = AppRoutes.BURGERZAKEN;
