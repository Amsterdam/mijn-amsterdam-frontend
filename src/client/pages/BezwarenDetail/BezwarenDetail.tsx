import { useParams } from 'react-router-dom';
import {
  useAppStateBagApi,
  useAppStateGetter,
  usePhoneScreen,
} from '../../hooks';
import {
  defaultDateFormat,
  isError,
  isLoading,
  uniqueArray,
} from '../../../universal/helpers';
import {
  ErrorAlert,
  ChapterIcon,
  DetailPage,
  InfoDetail,
  LoadingContent,
  PageContent,
  PageHeading,
} from '../../components';
import {
  AppRoutes,
  BagChapters,
  ChapterTitles,
} from '../../../universal/config';
import {
  InfoDetailGroup,
  InfoDetailHeading,
} from '../../components/InfoDetail/InfoDetail';
import BezwarenStatusLines from './BezwarenStatusLines';
import DocumentList from '../../components/DocumentList/DocumentList';
import styles from './BezwarenDetail.module.scss';
import { TextClamp } from '../../components/InfoDetail/TextClamp';
import { BFFApiUrls } from '../../config/api';
import { BezwaarDetail } from '../../../server/services/bezwaren/bezwaren';
import { BarConfig } from '../../components/LoadingContent/LoadingContent';
import { Heading } from '@amsterdam/design-system-react';

const loadingContentBarConfig2: BarConfig = [
  ['30rem', '4rem', '2rem'],
  ['30rem', '4rem', '2rem'],
  ['30rem', '4rem', '2rem'],
];

const BezwarenDetail = () => {
  const { BEZWAREN } = useAppStateGetter();
  const { uuid } = useParams<{ uuid: string }>();

  const bezwaar = BEZWAREN.content?.find((b) => b.uuid === uuid) ?? null;

  const [bezwaarDetail, api] = useAppStateBagApi<BezwaarDetail | null>({
    url: `${BFFApiUrls.BEZWAREN_DETAIL}/${uuid}`,
    bagChapter: BagChapters.BEZWAREN,
    key: uuid,
  });

  const noContent = !isLoading(BEZWAREN) && !bezwaar;
  const documents = bezwaarDetail?.documents ?? [];
  const statussen = bezwaarDetail?.statussen ?? [];
  const documentCategories = uniqueArray(
    documents.map((d) => d.dossiertype).filter(Boolean)
  ).sort();

  const isSmallScreen = usePhoneScreen();

  return (
    <DetailPage>
      <PageHeading
        icon={<ChapterIcon />}
        backLink={{
          to: AppRoutes.BEZWAREN,
          title: ChapterTitles.BEZWAREN,
        }}
        isLoading={isLoading(BEZWAREN)}
      >
        {bezwaar?.identificatie ?? 'Bezwaar'}
      </PageHeading>

      <PageContent>
        {(isError(BEZWAREN) || noContent || api.isError) && (
          <ErrorAlert>We kunnen op dit moment geen gegevens tonen.</ErrorAlert>
        )}
        {!!bezwaar && (
          <>
            {bezwaar.omschrijving && (
              <InfoDetail label="Onderwerp" value={bezwaar.omschrijving} />
            )}
            {bezwaar.toelichting && (
              <InfoDetail
                label="Reden bezwaar"
                value={
                  <TextClamp tagName="span" minHeight="100px" maxHeight="200px">
                    {bezwaar.toelichting}
                  </TextClamp>
                }
              />
            )}
            {bezwaar.primairbesluit && bezwaar.primairbesluitdatum && (
              <InfoDetailGroup>
                <InfoDetail
                  label="Besluit waartegen u bezwaar maakt"
                  value={bezwaar.primairbesluit}
                />
                <InfoDetail
                  label="Datum"
                  value={defaultDateFormat(bezwaar.primairbesluitdatum)}
                />
              </InfoDetailGroup>
            )}
            {bezwaar.einddatum && bezwaar.resultaat && (
              <InfoDetail label="Resultaat bezwaar" value={bezwaar.resultaat} />
            )}
            {documentCategories.length > 0 && (
              <>
                {documentCategories.map((category) => {
                  const docs = documents.filter(
                    (d) => d.dossiertype === category
                  );
                  return (
                    <InfoDetailGroup
                      key={category}
                      label={
                        <div className={styles.DocumentListHeader}>
                          <InfoDetailHeading
                            label={`Document${
                              documents.length > 1 ? 'en' : ''
                            } ${category.toLowerCase()}`}
                          />
                          {!isSmallScreen && (
                            <span className={styles.DocumentListHeader_Date}>
                              Datum
                            </span>
                          )}
                        </div>
                      }
                    >
                      <DocumentList documents={docs} showDatePublished />
                    </InfoDetailGroup>
                  );
                })}
              </>
            )}
          </>
        )}
      </PageContent>
      {api.isLoading && (
        <PageContent>
          <Heading level={4} size="level-4">
            Status
          </Heading>
          <LoadingContent barConfig={loadingContentBarConfig2} />
        </PageContent>
      )}
      {!!bezwaarDetail && !!bezwaar?.uuid && (
        <BezwarenStatusLines id={bezwaar.uuid} statussen={statussen} />
      )}
    </DetailPage>
  );
};

export default BezwarenDetail;
