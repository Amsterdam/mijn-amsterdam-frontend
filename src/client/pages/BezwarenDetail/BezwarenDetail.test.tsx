import { render } from '@testing-library/react';
import { generatePath } from 'react-router-dom';
import { MutableSnapshot } from 'recoil';
import { AppRoutes } from '../../../universal/config';
import { appStateAtom } from '../../hooks';
import MockApp from '../MockApp';
import BezwarenDetail from './BezwarenDetail';

const testState = {
  BEZWAREN: {
    content: [
      {
        identificatie: 'BI.21.014121.001',
        uuid: '68cdd171-b4fd-44cc-a4d3-06b77341f20a',
        ontvangstdatum: '2023-04-25',
        bezwaarnummer: 'BI.21.014121.001',
        zaakkenmerk: 'ghi.abc.def',
        omschrijving: 'Korte omschrijving van het bezwaar',
        toelichting:
          'Lange uitleg over het bezwaar. Kan dus veel tekst hebben want is vrije invoer.',
        status: null,
        statussen: [
          {
            uuid: 'b62fdaa9-f7ec-45d1-b23c-7f36fa00b393',
            datum: '2023-03-29T10:00:00+02:00',
            statustoelichting: 'Ontvangen',
          },
          {
            uuid: '50dcd25b-a826-422b-b51c-9049dda9600c',
            datum: '2023-03-30T02:53:00+02:00',
            statustoelichting: 'In behandeling',
          },
          {
            uuid: '00000000-0000-0000-0000-000000000000',
            datum: '',
            statustoelichting: 'Afgehandeld',
          },
        ],
        datumbesluit: '01-05-2021',
        datumIntrekking: null,
        einddatum: null,
        primairbesluit: 'F2021.0008',
        primairbesluitdatum: '01-05-2021',
        resultaat: 'Niet-ontvankelijk',
        documenten: [
          {
            id: 'e6ed38c3-a44a-4c16-97c1-89d7ebfca095',
            title: 'Constructie.pdf',
            datePublished: '23 maart 2023',
            url: '#',
          },
          {
            id: 'a9efe7c9-b2d2-4162-bb7b-00189f4c60d0',
            title: 'Test document 01.pdf',
            datePublished: '23 maart 2023',
            url: '#',
          },
          {
            id: '42034d2a-921e-4466-a205-c0356fa6be55',
            title: 'Concept besluit.pdf',
            datePublished: '23 maart 2023',
            url: '#',
          },
          {
            id: 'd5aa70d6-f54b-4bd9-a6c1-42a9fd9acf9c',
            title: 'BZ020001.pdf',
            datePublished: '23 maart 2023',
            url: '#',
          },
        ],
        link: {
          title: 'Bekijk details',
          to: '/bezwaren/68cdd171-b4fd-44cc-a4d3-06b77341f20a',
        },
      },
    ],
    status: 'OK',
  },
};

function initializeState(testState: any) {
  return (snapshot: MutableSnapshot) => snapshot.set(appStateAtom, testState);
}

function setupTestComponent(id: string) {
  const routeEntry = generatePath(AppRoutes['BEZWAREN/DETAIL'], {
    uuid: id,
  });
  const routePath = AppRoutes['BEZWAREN/DETAIL'];

  return () => (
    <MockApp
      routeEntry={routeEntry}
      routePath={routePath}
      component={BezwarenDetail}
      initializeState={initializeState(testState)}
    />
  );
}

describe('BezwarenDetail', () => {
  describe('No result', () => {
    const Component = setupTestComponent('abc');

    it('Matches the Full Page snapshot', () => {
      const { asFragment } = render(<Component />);
      expect(asFragment()).toMatchSnapshot();
    });
  });

  describe('With result', () => {
    const Component = setupTestComponent(
      '68cdd171-b4fd-44cc-a4d3-06b77341f20a'
    );

    it('Matches the Full Page snapshot', () => {
      const { asFragment } = render(<Component />);
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
