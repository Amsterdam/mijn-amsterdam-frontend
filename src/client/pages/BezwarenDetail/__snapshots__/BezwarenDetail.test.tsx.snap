// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`BezwarenDetail > No result > Matches the Full Page snapshot 1`] = `
<DocumentFragment>
  <main
    class="_Page_47d7c0 _DetailPage_47d7c0"
  >
    <header
      class="_PageHeading_3b337d _hasIcon_3b337d _hasBackLink_3b337d"
    >
      <span
        aria-hidden="true"
        class="_Icon_3b337d"
      >
        <svg
          aria-label="Bezwaren"
          class="_ChapterIcon_3ea2dd"
          fill="black"
          height="32"
          viewBox="0 0 32 32"
          width="32"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M15 19H26V21H15V19Z"
          />
          <path
            d="M26 23H15V25H26V23Z"
          />
          <path
            clip-rule="evenodd"
            d="M9 18C4.02945 18 0 13.9706 0 9C0 4.02942 4.02945 0 9 0C12.1204 0 14.8699 1.58801 16.4845 4H32V32H9V18ZM17.777 11C17.9229 10.3568 18 9.68738 18 9C18 8.66199 17.9814 8.32831 17.9451 8H28V28H13V17.0645C13.7281 16.7026 14.4003 16.2451 15 15.7083V17H26V15H15.7083C16.2451 14.4003 16.7026 13.7281 17.0645 13H26V11H17.777Z"
            fill-rule="evenodd"
          />
          <rect
            fill="#D9D9D9"
            height="7"
            width="2"
            x="8"
            y="3"
          />
          <rect
            fill="#D9D9D9"
            height="2"
            transform="matrix(1 0 0 -1 8 14)"
            width="2"
          />
        </svg>
      </span>
      <div
        class="_HeadingInner_3b337d"
      >
        <a
          class="_Button_c3ab45 _Button__plain_c3ab45 _Button__lean_c3ab45 _Linkd_c3ab45 _BackLink_3b337d"
          href="/bezwaren"
        >
          <svg
            aria-hidden="true"
            class="_Icon_c3ab45 _Icon__left_c3ab45"
            fill="#000"
            focusable="false"
            height="14px"
            viewBox="0 0 32 32"
            width="14px"
            xmlns="http://www.w3.org/2000/svg"
            xmlns:xlink="http://www.w3.org/1999/xlink"
          >
            <title>
              chevron-links-icoon
            </title>
            <polygon
              points="22.8571429 32 6.85714286 16 22.8571429 0 25.7568858 2.91002571 12.6771943 16 25.7568858 29.0899743"
            />
          </svg>
          Bezwaren
        </a>
        <h2
          class="ams-heading ams-heading--1"
        >
          Bezwaar
        </h2>
      </div>
    </header>
    <div
      class="_PageContent_47d7c0"
    >
      <section
        class="ams-alert ams-alert--error"
      >
        <div
          class="ams-alert__icon"
        >
          <span
            class="ams-icon ams-icon--size-4"
          >
            <svg
              aria-hidden="true"
              focusable="false"
              viewBox="0 0 36 36"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M33.63 30.9H2.37L18 3zm-24.44-4h17.62L18 11.19z"
              />
              <path
                d="M17 16.99h2v4.95h-2zm0 5.95h2v1.95h-2z"
              />
            </svg>
          </span>
        </div>
        <div
          class="ams-alert__content"
        >
          <h2
            class="ams-heading ams-heading--4"
          >
            Foutmelding
          </h2>
          <p
            class="ams-paragraph"
          >
            We kunnen op dit moment geen gegevens tonen.
          </p>
        </div>
      </section>
    </div>
  </main>
</DocumentFragment>
`;

exports[`BezwarenDetail > With result > Matches the Full Page snapshot 1`] = `
<DocumentFragment>
  <main
    class="_Page_47d7c0 _DetailPage_47d7c0"
  >
    <header
      class="_PageHeading_3b337d _hasIcon_3b337d _hasBackLink_3b337d"
    >
      <span
        aria-hidden="true"
        class="_Icon_3b337d"
      >
        <svg
          aria-label="Bezwaren"
          class="_ChapterIcon_3ea2dd"
          fill="black"
          height="32"
          viewBox="0 0 32 32"
          width="32"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M15 19H26V21H15V19Z"
          />
          <path
            d="M26 23H15V25H26V23Z"
          />
          <path
            clip-rule="evenodd"
            d="M9 18C4.02945 18 0 13.9706 0 9C0 4.02942 4.02945 0 9 0C12.1204 0 14.8699 1.58801 16.4845 4H32V32H9V18ZM17.777 11C17.9229 10.3568 18 9.68738 18 9C18 8.66199 17.9814 8.32831 17.9451 8H28V28H13V17.0645C13.7281 16.7026 14.4003 16.2451 15 15.7083V17H26V15H15.7083C16.2451 14.4003 16.7026 13.7281 17.0645 13H26V11H17.777Z"
            fill-rule="evenodd"
          />
          <rect
            fill="#D9D9D9"
            height="7"
            width="2"
            x="8"
            y="3"
          />
          <rect
            fill="#D9D9D9"
            height="2"
            transform="matrix(1 0 0 -1 8 14)"
            width="2"
          />
        </svg>
      </span>
      <div
        class="_HeadingInner_3b337d"
      >
        <a
          class="_Button_c3ab45 _Button__plain_c3ab45 _Button__lean_c3ab45 _Linkd_c3ab45 _BackLink_3b337d"
          href="/bezwaren"
        >
          <svg
            aria-hidden="true"
            class="_Icon_c3ab45 _Icon__left_c3ab45"
            fill="#000"
            focusable="false"
            height="14px"
            viewBox="0 0 32 32"
            width="14px"
            xmlns="http://www.w3.org/2000/svg"
            xmlns:xlink="http://www.w3.org/1999/xlink"
          >
            <title>
              chevron-links-icoon
            </title>
            <polygon
              points="22.8571429 32 6.85714286 16 22.8571429 0 25.7568858 2.91002571 12.6771943 16 25.7568858 29.0899743"
            />
          </svg>
          Bezwaren
        </a>
        <h2
          class="ams-heading ams-heading--1"
        >
          BI.21.014121.001
        </h2>
      </div>
    </header>
    <div
      class="_PageContent_47d7c0"
    >
      <div
        class="_InfoDetail_6f1df2"
      >
        <h3
          class="ams-heading ams-heading--4 _Label_6f1df2"
        >
          Onderwerp
        </h3>
        <p
          class="_Value_6f1df2"
        >
          Korte omschrijving van het bezwaar
        </p>
      </div>
      <div
        class="_InfoDetail_6f1df2"
      >
        <h3
          class="ams-heading ams-heading--4 _Label_6f1df2"
        >
          Reden bezwaar
        </h3>
        <p
          class="_Value_6f1df2"
        >
          Lange uitleg over het bezwaar. Kan dus veel tekst hebben want is vrije invoer.
        </p>
      </div>
      <div
        class="_InfoDetailGroup_6f1df2"
      >
        <div
          class="_InfoDetailGroupContent_6f1df2"
        >
          <div
            class="_InfoDetail_6f1df2"
          >
            <h3
              class="ams-heading ams-heading--4 _Label_6f1df2"
            >
              Besluit waartegen u bezwaar maakt
            </h3>
            <p
              class="_Value_6f1df2"
            >
              F2021.0008
            </p>
          </div>
          <div
            class="_InfoDetail_6f1df2"
          >
            <h3
              class="ams-heading ams-heading--4 _Label_6f1df2"
            >
              Datum
            </h3>
            <p
              class="_Value_6f1df2"
            >
              01-05-2021
            </p>
          </div>
        </div>
      </div>
    </div>
    <div
      class="_PageContent_47d7c0"
    >
      <div
        class="_InfoDetailGroup_6f1df2"
      >
        <div
          class="_DocumentListHeader_839df0"
        >
          <h3
            class="ams-heading ams-heading--4 _Label_6f1df2"
          >
            Documenten online aangeleverd
          </h3>
          <span
            class="_DocumentListHeader_Date_839df0"
          >
            Datum
          </span>
        </div>
        <div
          class="_InfoDetailGroupContent_6f1df2"
        >
          <ul
            class="_DocumentList_7ed0ad"
          >
            <li
              class="_DocumentListItem_7ed0ad"
            >
              <span
                class="_DocumentLinkWrap_81c3b4"
              >
                <span
                  class="_DownloadIcon_81c3b4"
                >
                  <svg
                    aria-hidden="true"
                    focusable="false"
                    height="14"
                    viewBox="0 0 32 32"
                    width="14"
                    xmlns="http://www.w3.org/2000/svg"
                    xmlns:xlink="http://www.w3.org/1999/xlink"
                  >
                    <title>
                      download-icoon
                    </title>
                    <path
                      d="M0,27.9962465 L32,27.9962465 L32,32 L0,32 L0,27.9962465 Z M22.36,14.0131373 L25.19,16.8357835 L16,26.0444166 L6.81,16.8357835 L9.64,14.0131373 L14,18.3772287 L14,-3.55271368e-15 L18,-3.55271368e-15 L18,18.3772287 L22.36,14.0131373 Z"
                    />
                  </svg>
                </span>
                <a
                  class="_Button_c3ab45 _Button__plain_c3ab45 _Button__lean_c3ab45 _Linkd_c3ab45 _DocumentLink_81c3b4"
                  href="http://localhost:5000/api/v1/services/bezwaren/d0fe0ace-6f53-4342-b978-1cf12f8675be/attachments"
                  rel="external noopener noreferrer"
                >
                  connector.docx
                </a>
              </span>
              <time
                class="_DocumentListItem_Date_7ed0ad"
              >
                30 augustus 2023
              </time>
            </li>
            <li
              class="_DocumentListItem_7ed0ad"
            >
              <span
                class="_DocumentLinkWrap_81c3b4"
              >
                <span
                  class="_DownloadIcon_81c3b4"
                >
                  <svg
                    aria-hidden="true"
                    focusable="false"
                    height="14"
                    viewBox="0 0 32 32"
                    width="14"
                    xmlns="http://www.w3.org/2000/svg"
                    xmlns:xlink="http://www.w3.org/1999/xlink"
                  >
                    <title>
                      download-icoon
                    </title>
                    <path
                      d="M0,27.9962465 L32,27.9962465 L32,32 L0,32 L0,27.9962465 Z M22.36,14.0131373 L25.19,16.8357835 L16,26.0444166 L6.81,16.8357835 L9.64,14.0131373 L14,18.3772287 L14,-3.55271368e-15 L18,-3.55271368e-15 L18,18.3772287 L22.36,14.0131373 Z"
                    />
                  </svg>
                </span>
                <a
                  class="_Button_c3ab45 _Button__plain_c3ab45 _Button__lean_c3ab45 _Linkd_c3ab45 _DocumentLink_81c3b4"
                  href="http://localhost:5000/api/v1/services/bezwaren/736ae47e-e703-4238-a664-100cde4c90a1/attachments"
                  rel="external noopener noreferrer"
                >
                  Bezwaar_JB20230049.pdf
                </a>
              </span>
              <time
                class="_DocumentListItem_Date_7ed0ad"
              >
                30 augustus 2023
              </time>
            </li>
          </ul>
        </div>
      </div>
      <div
        class="_InfoDetailGroup_6f1df2"
      >
        <div
          class="_DocumentListHeader_839df0"
        >
          <h3
            class="ams-heading ams-heading--4 _Label_6f1df2"
          >
            Documenten online besluitvorming
          </h3>
          <span
            class="_DocumentListHeader_Date_839df0"
          >
            Datum
          </span>
        </div>
        <div
          class="_InfoDetailGroupContent_6f1df2"
        >
          <ul
            class="_DocumentList_7ed0ad"
          >
            <li
              class="_DocumentListItem_7ed0ad"
            >
              <span
                class="_DocumentLinkWrap_81c3b4"
              >
                <span
                  class="_DownloadIcon_81c3b4"
                >
                  <svg
                    aria-hidden="true"
                    focusable="false"
                    height="14"
                    viewBox="0 0 32 32"
                    width="14"
                    xmlns="http://www.w3.org/2000/svg"
                    xmlns:xlink="http://www.w3.org/1999/xlink"
                  >
                    <title>
                      download-icoon
                    </title>
                    <path
                      d="M0,27.9962465 L32,27.9962465 L32,32 L0,32 L0,27.9962465 Z M22.36,14.0131373 L25.19,16.8357835 L16,26.0444166 L6.81,16.8357835 L9.64,14.0131373 L14,18.3772287 L14,-3.55271368e-15 L18,-3.55271368e-15 L18,18.3772287 L22.36,14.0131373 Z"
                    />
                  </svg>
                </span>
                <a
                  class="_Button_c3ab45 _Button__plain_c3ab45 _Button__lean_c3ab45 _Linkd_c3ab45 _DocumentLink_81c3b4"
                  href="http://localhost:5000/api/v1/services/bezwaren/d0fe0ace-6f53-4342-b978-1cf12f8675be/attachments"
                  rel="external noopener noreferrer"
                >
                  connector.docx
                </a>
              </span>
              <time
                class="_DocumentListItem_Date_7ed0ad"
              >
                30 augustus 2023
              </time>
            </li>
          </ul>
        </div>
      </div>
      <div
        class="_InfoDetailGroup_6f1df2"
      >
        <div
          class="_DocumentListHeader_839df0"
        >
          <h3
            class="ams-heading ams-heading--4 _Label_6f1df2"
          >
            Documenten online procesdossier
          </h3>
          <span
            class="_DocumentListHeader_Date_839df0"
          >
            Datum
          </span>
        </div>
        <div
          class="_InfoDetailGroupContent_6f1df2"
        >
          <ul
            class="_DocumentList_7ed0ad"
          >
            <li
              class="_DocumentListItem_7ed0ad"
            >
              <span
                class="_DocumentLinkWrap_81c3b4"
              >
                <span
                  class="_DownloadIcon_81c3b4"
                >
                  <svg
                    aria-hidden="true"
                    focusable="false"
                    height="14"
                    viewBox="0 0 32 32"
                    width="14"
                    xmlns="http://www.w3.org/2000/svg"
                    xmlns:xlink="http://www.w3.org/1999/xlink"
                  >
                    <title>
                      download-icoon
                    </title>
                    <path
                      d="M0,27.9962465 L32,27.9962465 L32,32 L0,32 L0,27.9962465 Z M22.36,14.0131373 L25.19,16.8357835 L16,26.0444166 L6.81,16.8357835 L9.64,14.0131373 L14,18.3772287 L14,-3.55271368e-15 L18,-3.55271368e-15 L18,18.3772287 L22.36,14.0131373 Z"
                    />
                  </svg>
                </span>
                <a
                  class="_Button_c3ab45 _Button__plain_c3ab45 _Button__lean_c3ab45 _Linkd_c3ab45 _DocumentLink_81c3b4"
                  href="http://localhost:5000/api/v1/services/bezwaren/736ae47e-e703-4238-a664-100cde4c90a1/attachments"
                  rel="external noopener noreferrer"
                >
                  Bezwaar_JB20230049.pdf
                </a>
              </span>
              <time
                class="_DocumentListItem_Date_7ed0ad"
              >
                30 augustus 2023
              </time>
            </li>
          </ul>
        </div>
      </div>
    </div>
    <div
      class="_StatusLine_a12ff2"
    >
      <h4
        class="ams-heading ams-heading--4 _ListHeading_a12ff2"
      >
        Status
      </h4>
      <ul
        class="_List_a12ff2"
      >
        <li
          class="_ListItem_a12ff2"
        >
          <div
            class="_ListItemInner_a12ff2"
          >
            <div
              aria-label="Vinkje, processtap gereed"
              class="_StatusConnection_a12ff2"
            >
              <span
                class="_ConnectLine_a12ff2 _ConnectLine--start-checked_a12ff2"
              />
              <span
                class="_Checkmark_a12ff2 _Checkmark--checked_a12ff2"
              />
            </div>
            <div
              class="_Panel_a12ff2 _Panel--status_a12ff2"
            >
              <strong
                class="_StatusTitle_a12ff2"
              >
                Ontvangen
              </strong>
              <time
                class="_StatusDate_a12ff2"
              >
                29 maart 2023
              </time>
            </div>
            <div
              class="_Panel_a12ff2 _Panel--description_a12ff2"
            >
              <div
                class="_PanelContent_a12ff2"
              />
            </div>
            <div
              class="_Panel_a12ff2 _Panel--documents_a12ff2"
            />
          </div>
        </li>
        <li
          class="_ListItem_a12ff2 _ListItem--highlight_a12ff2"
        >
          <div
            class="_ListItemInner_a12ff2"
          >
            <div
              aria-label="Vinkje, processtap gereed"
              class="_StatusConnection_a12ff2"
            >
              <span
                class="_ConnectLine_a12ff2 _ConnectLine--middle-checked-next_a12ff2"
              />
              <span
                class="_Checkmark_a12ff2 _Checkmark--active_a12ff2 _Checkmark--checked_a12ff2"
              />
            </div>
            <div
              class="_Panel_a12ff2 _Panel--status_a12ff2"
            >
              <strong
                class="_StatusTitle_a12ff2"
              >
                In behandeling
              </strong>
              <time
                class="_StatusDate_a12ff2"
              >
                30 maart 2023
              </time>
            </div>
            <div
              class="_Panel_a12ff2 _Panel--description_a12ff2"
            >
              <div
                class="_PanelContent_a12ff2"
              />
            </div>
            <div
              class="_Panel_a12ff2 _Panel--documents_a12ff2"
            />
          </div>
        </li>
        <li
          class="_ListItem_a12ff2"
        >
          <div
            class="_ListItemInner_a12ff2"
          >
            <div
              aria-label="Vinkje, processtap gereed"
              class="_StatusConnection_a12ff2"
            >
              <span
                class="_ConnectLine_a12ff2 _ConnectLine--end_a12ff2"
              />
              <span
                class="_Checkmark_a12ff2"
              />
            </div>
            <div
              class="_Panel_a12ff2 _Panel--status_a12ff2"
            >
              <strong
                class="_StatusTitle_a12ff2"
              >
                Afgehandeld
              </strong>
            </div>
            <div
              class="_Panel_a12ff2 _Panel--description_a12ff2"
            >
              <div
                class="_PanelContent_a12ff2"
              />
            </div>
            <div
              class="_Panel_a12ff2 _Panel--documents_a12ff2"
            />
          </div>
        </li>
      </ul>
    </div>
  </main>
</DocumentFragment>
`;
